src/components/organisms/AssetActions/Edit/index.tsx

```diff
@@ -67,22 +68,43 @@ export default function Edit({
     resetForm: () => void
   ) {
     try {
-      const ddo = await ocean.assets.editMetadata(
-        did,
-        { title: values.name, description: values.description },
-        account
-      )
+      const ddoEditedMetdata = await ocean.assets.editMetadata(ddo, {
+        title: values.name,
+        description: values.description
+      })
+      if (!ddoEditedMetdata) {
+        setError(content.form.error)
+        Logger.error(content.form.error)
+        return
+      }
 
-      // Edit failed
-      if (!ddo) {
+      const service = ddo.findServiceByType('access')
+      const timeout = mapTimeoutStringToSeconds(values.timeout)
+      const ddoEditedTimeout = await ocean.assets.editServiceTimeout(
+        ddo,
```

Comment:
Shouldn't it be `ddoEditedMetdata` instead of `ddo`, you ignore the edits to title and description this way. Or am i misunderstanding

Reply:
Yes it should, I'll change that, strangely it works with `ddo` to

Reply:
Ah, i see, the return value is kind of useless, ocean.js uses the reference so it will change `ddo`, and thus working as you said

Reply:
Yas that's it